src/main/resources/api/openapi.yaml

```diff
@@ -1884,6 +1885,13 @@ components:
           minimum: 0
           example: 667
           nullable: true
+        maxPeerHeight:
+          type: integer
+          format: int32
+          description: Can be 'null' if state is empty (no peer connected since node launch)
```

Comment:
Explanation of peerHeight would be good here

Reply:
added